ldi-orchestrator/ldio-connectors/ldio-kakfa-in/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/KafkaInAuthStrategyTest.java

```diff
@@ -0,0 +1,22 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.config;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class KafkaInAuthStrategyTest {
+
+    @ParameterizedTest
+    @EnumSource(KafkaInAuthStrategy.class)
+    void from_ShouldReturnAValueForExistingAuthStrategies(KafkaInAuthStrategy authStrategy) {
```

Comment:
Took me a while to figure out what the "from" was for.. I understand you just want to reuse the method name, but it might be confusing when reading the error IF a test should fail 😄

Reply:
I removed the from :)